lib/perl/Genome/InstrumentData/Solexa.pm

```diff
@@ -1253,10 +1253,9 @@ sub get_default_alignment_results {
 
     unless(@alignment_results) {
         # Get alignment results for this inst data and the default aligner name
-        my $pp = Genome::ProcessingProfile::ReferenceAlignment->default_profile;
         @alignment_results = Genome::InstrumentData::AlignmentResult->get(
             instrument_data_id => $self->id,
-            aligner_name       => $pp->read_aligner_name,
+            aligner_name       => 'bwa',
```

Comment:
just feel a little odd for this hard coded "bwa" here.  Is the old alignment QC metric still being used after the new QC config is in place ? If so, probably it's better to get aligner_name from a processing_profile that is listed in a "default" analysis menu item like "Human Reference Alignment".